Meadowlark-js/packages/meadowlark-core/src/validation/ResourceSchemaValidation.ts

```diff
@@ -18,8 +18,9 @@ export type ResourceSchemaValidators = {
   updateValidator: ValidateFunction;
 };
 
-function initializeAjv(): Ajv {
-  const removeAdditional = false; // TODO: replace on merge with RND-651
+function initializeAjv(isQueryParameterValidator: boolean): Ajv {
+  // A query parameter validator cannot have additional properties
+  const removeAdditional = isQueryParameterValidator ? false : getBooleanFromEnvironment('ALLOW_OVERPOSTING', false);
```

Comment:
Interestingly, the ODS/API does not reject spurious query string values. However, I agree with the approach here in Meadowlark. We should let people know that their query string was invalid.